api/api.go

```diff
@@ -360,6 +360,14 @@ func (api *Server) SendAction(ctx context.Context, in *iotexapi.SendActionReques
 	if err = selp.LoadProto(in.Action); err != nil {
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
+
+	// reject action if chainID is not matched at JutlandHeight
+	if api.cfg.Genesis.Blockchain.IsJutland(api.bc.TipHeight()) {
+		if api.bc.ChainID() != in.GetAction().Core.GetChainID() {
+			return nil, status.Error(codes.InvalidArgument, "ChainID does not match")
+		}
+	}
```

Comment:
we need a period to tolerant it

Reply:
gated by Jutland(), but will need to change to next height